models/migrations/v93.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"code.gitea.io/gitea/modules/util"
+
+	"github.com/go-xorm/xorm"
+)
+
+func addLanguageStats(x *xorm.Engine) error {
+	// LanguageStat see models/repo_language_stats.go
+	type LanguageStat struct {
+		ID          int64 `xorm:"pk autoincr"`
+		RepoID      int64 `xorm:"UNIQUE(s) INDEX NOT NULL"`
+		CommitID    string
+		IsPrimary   bool
+		Language    string         `xorm:"VARCHAR(30) UNIQUE(s) INDEX NOT NULL"`
+		Percentage  float32        `xorm:"NUMERIC(4,2) NOT NULL DEFAULT 0"`
+		Color       string         `xorm:"-"`
+		CreatedUnix util.TimeStamp `xorm:"INDEX CREATED"`
```

Comment:
I think you need to use `code.gitea.io/gitea/modules/timeutil` to reference `TimeStamp`